kytos/core/rest_api.py

```diff
@@ -149,3 +145,10 @@ def body(self) -> Optional[str]:
         if body is None:
             return None
         return body.decode("utf-8")
+
+    @property
+    def mimetype(self) -> str:
+        return (
+            self.request.headers.get("Content-Type")
+            or "application/octet-stream"
+        )
```

Comment:
`openapi-core==0.19.0` has some issues including:
- Not differentiating between numeric string and integers
- Broken spec validation can cause to accept any request
From my tests `openapi-core==0.18.2` is more stable but this property which causes some errors. Brought this from current request [implementation](https://github.com/python-openapi/openapi-core/blob/0.19.0/openapi_core/contrib/starlette/requests.py).

Reply:
Created this [issue](https://github.com/kytos-ng/kytos/issues/466). What does issue upstream mean?